processors/contact/class-contact-processor.php

```diff
@@ -331,10 +331,11 @@ public function process_address( $config, $form, $transient, &$form_values ){
 				// Pass address ID if we got one
 				if ( isset( $address ) && is_array( $address ) ) {
 					$form_values['civicrm_address']['id'] = $address['id']; // Address ID
-				} else {
-					$form_values['civicrm_address']['location_type_id'] = $config['civicrm_address']['location_type_id'];
```

Comment:
@kcristiano These two lines are now redundant and could be removed.

Reply:
I agree, they are being removed.

Reply:
Doh, silly me!